api/src/DTO/Invitation.php

```diff
@@ -12,11 +12,18 @@
  * already have an account.
  */
 #[ApiResource(
-    collectionOperations: [],
+    collectionOperations: [
+        'get' => [
+            'security' => 'false',
```

Comment:
Würde es funktionieren, hier eine 404Response zurückzugeben, wie es in https://api-platform.com/docs/core/operations/#enabling-and-disabling-operations beschrieben ist?

Reply:
Aus https://github.com/ecamp/ecamp3/pull/2245/commits/24c1a904e7555ac0d6a345eb3b3a3766787715ed

For /invitations, we need to add a dummy get collection operation.
For /invitations{/id}, we also need to define a get operation.
We cannot use a dummy get operation,
because then the self links would look like this:
'self' =>
     array (
-      'href' => '/invitations/myInviteKey2/find',
+      'href' => '/invitations/myInviteKey2',
     ),
If parameters with different names had been defined
in the same path segment, it would not make sense
to expose all the possible names.
Thus, we use the existing functionality of the UriTemplateFactory
to add an {/id} template parameter if the get item operation is defined.

`$this->resourceMetadataFactory->create($resourceClass);` in UriTemplateFactory gibt nichts (oder einen Fehler, bin mir nicht mehr sicher) zurück, wenn die Ressource keine Collection Operation hat

Reply:
A I see. Klingt nach etwas was in der UriTemplateFactory gefixt werden könnte.
Anyways, die Technik die ich verlinke beschreibt nicht (ausschliesslich), die Operation ganz wegzulassen, sondern auch wie man eine GET Item Operation hinschreiben kann, die dann immer einen 404 zurückgibt. Das ist in gewissen edge cases nötig für den Symfony Router, bei uns würde es aber nur den Status von 403 auf 404 ändern, plus der fake-endpoint würde hoffentlich in den openapi docs herausgefiltert.
Ich probiere selber mal aus obs sich so verhält wie ich annehme und mache ggf. einen PR.